### DISCUSSION TITLE
［求助］之前不知道怎么进空洞里了，遇到了奇怪的以骸

### DISCUSSION THREAD

**Discussion by Huanshenxuanlong** [share121/inter-knot#2376] [其他]
![e95fec0a1fc133c396d043a319abb03a](https://github.com/user-attachments/assets/8b55ea5c-a22b-4acc-bf38-e8448bcb96dc)
真是邪门了，那些以骸不伤人顶着个骰子还有麻将脑袋，嘴里说着什么：“信了，信了，他真的信了！他信这是现实世界了！”然后对着一个跪在地上喊“我分不清，我是真的分不清啊！”的穿着红衣服的帅哥说。
这是什么以骸啊？还会说人话，要报告治安局吗？

**Comment by ConsistDust3838** [comment]
休想骗我，你个坐忘道

**Comment by Yith123** [comment]
这个治安局不管，你可以找找本地帮派，比如监天司，坐忘道什么的
